packages/core/botpress/src/content/service.js

```diff
@@ -314,13 +314,14 @@ module.exports = async ({ botfile, projectLocation, logger, ghostManager }) => {
     const item = { formData, ...(await fillComputedProps(category, formData)) }
     const body = transformItemApiToDb(item)
 
+    const newItemId = itemId || getNewItemId(category)
```

Comment:
This looks wrong, i.e. the next condition will always fail. `itemId` will always evaluate to true?

Reply:
@slvnperron no, `itemId === undefined`

Reply:
Right, when it's undefined it's creation of the new content item.
It would be better to name it differently though like I would do

```
const isNewItemCreation = !!itemId
if (!isNewItemCreation) {
  itemId = getNewItemId(category)
}
```

Then use `isNewItemCreation` in the condition below, and `itemId` everywhere

Reply:
I think it would be better use `!`
```const isNewItemCreation = !itemId
if (isNewItemCreation) {
  itemId = getNewItemId(category)
}```
because when we have new item `isNewItemCreation === false`
what do you think @emirotin ?